text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/tag/standard/StandardTags.java

```diff
@@ -214,6 +216,26 @@ public static TagResolver transition() {
     return NewlineTag.RESOLVER;
   }
 
+  /**
+   * Get a resolver for the {@value ProgressTag#PROGRESS} tag.
+   *
+   * @return a resolver for the {@value ProgressTag#PROGRESS} tag
+   * @since 4.11.0
+   */
+  public static @NotNull TagResolver progress() {
+    return ProgressTag.RESOLVER;
+  }
+
+  /**
+   * Get a resolver for the {@value RepeatTag#REPEAT} tag.
+   *
+   * @return a resolver for the {@value RepeatTag#REPEAT} tag
+   * @since 4.11.0
```

Comment:
```suggestion
   * @since 4.13.0
```

unlikely to make it in any earlier than that